### PR TITLE
updating readme and scripts to upload

### PR DESCRIPTION
**Pull request by sumitarora**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,12 @@ Batarangle is a Google Chrome Dev Tools extension for debugging Angular 2 applic
 
 If you want to contribute or need help getting started, [join us on Slack](https://batarangle-slack.herokuapp.com).
 
+## Getting Extension
```
Comment (igor-ka) [inline README.md]: @sumitarora can we mention that there are 2 ways of getting it?
Specifically, 1) If you just want to use it, download it from batarangle.io 
2) for more adventurous people there are instructions on how to build and pack it yourself. (we should mention that our CI process creates a package with the latests master on batarangle.io)

**Comment by igor-ka** [comment]
@sumitarora since do not need to modify the application level code anymore in order to get Batarangle to work (as ng.probe is available in angular 2 dev mode) we should remove this from limitations then (or wherever else we are mentioning it), but mention somewhere that angular 2 needs to run in dev mode (it's default mode, please confirm) in order for batarangle to work.

**Comment by sumitarora** [comment]
@igor-ka done

**Comment by igor-ka** [comment]
@sumitarora LGTM :shipit:

**Pull request merged**